<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>891</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="AttendanceLAyout">
    <item>
     <widget class="QWidget" name="AttendanceWidget" native="true">
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>861</width>
         <height>391</height>
        </rect>
       </property>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Student ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Year</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Section</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Course</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Gender/Sex</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Attendance Status</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Time IN</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Time OUT</string>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>291</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Event: Event Name</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>Filter</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>530</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Go Back</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
